src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -51,6 +56,24 @@ def __call__(self, organization, event):
                 author_email = u'{}@localhost'.format(
                     author_email[:65],
                 )
+            # try to figure out who anonymous emails are
+            elif '@users.noreply.github.com' in author_email:
+                gh_username = author_email.split('@')[0]
+                try:
+                    gh_user = client.request_no_auth('GET', '/users/%s' % gh_username)
+                except ApiError as exc:
+                    logger.exception(six.text_type(exc))
+                else:
+                    try:
+                        user = User.objects.filter(
```

Comment:
Is it possible for us to match more than 1 user? If so, what's our behavior here? Seems bad if we just pick the first one.